app/api/server/v1/users.js

```diff
@@ -283,7 +283,9 @@ API.v1.addRoute('users.list', { authRequired: true }, {
 	},
 });
 
-API.v1.addRoute('users.register', { authRequired: false }, {
+API.v1.addRoute('users.register', { authRequired: false,
```

Comment:
Here you will need to keep the object reference and update the values on setting changes, after each change you need to call the API method `reloadRoutesToRefreshRateLimiter` to refresh the limiters